assets/chars.json

```diff
@@ -417,54 +417,79 @@
   },
   {
     "name": "Hyunjin Choi",
-    "age": "??",
+    "age": 20,
     "description": "Hyunjin is a Tech specialist hired after Keitaro's adventures to help bring the camp into the modern era. He is going outside of his comfort zone by going to the camp.",
-    "birthday": "??",
-    "animal": "??",
-    "color": "#3C5396",
-    "height": "??",
-    "weight": "??",
-    "blood_type": "??",
+    "birthday": "February 6",
+    "animal": "Owl",
+    "color": "#3a5396",
+    "height": "170 cm (5'7\")",
+    "weight": "55 kg (121 lbs)",
+    "blood_type": "O",
     "emoteId": "909168197691904030",
     "cuteEmoteId": "909168197691904030"
   },
   {
     "name": "Emilia Komarova",
-    "age": "??",
+    "age": 29,
     "description": "Emilia has been sent out by Clermont Inc. to make sure that Camp Buddy meets all of the requirements set by the sponsorship.",
-    "birthday": "??",
-    "animal": "??",
-    "color": "#1FD3A9",
-    "height": "??",
-    "weight": "??",
-    "blood_type": "??",
+    "birthday": "December 5",
+    "animal": "Mosquito",
+    "color": "#20d3a9",
+    "height": "170 cm (5'7\")",
+    "weight": "59 kg (130 lbs)",
+    "blood_type": "B",
     "emoteId": "909168184551157830",
     "cuteEmoteId": "909168184551157830"
   },
   {
     "name": "Lloyd Sirus",
-    "age": "??",
+    "age": 30,
     "description": "Lloyd is a high-energy architect who's often teased for his short stature. He can be eccentric at times when it comes to his peculiar hobbies.",
-    "birthday": "??",
-    "animal": "??",
+    "birthday": "March 21",
+    "animal": "Hamster",
     "color": "#7CA2D1",
-    "height": "??",
-    "weight": "??",
-    "blood_type": "??",
+    "height": "158 cm (5'2\")",
+    "weight": "57 kg (125 lbs)",
+    "blood_type": "B",
     "emoteId": "909166826234212412",
     "cuteEmoteId": "909166826234212412"
```

Comment:
Add emote id of lloyd cure from CB server

Reply:
Duncan and myself checked the IDs, they should be correct

Reply:
These IDs are for the unknown chibi emotes, they are only placeholders

Reply:
Are you sure? These are the ID's we have on the CB server and they line up with what's here:

<:HyunjinCute:909168197691904030>
<:EmiliaCute:909168184551157830>
<:LloydCute:909166826234212412>
<:DariusCute:909168172073103361>